core/io/resource_format_binary.cpp

```diff
@@ -674,7 +674,7 @@ Error ResourceInteractiveLoaderBinary::poll() {
 
 	RES res = RES(r);
 
-	r->set_path(path);
+	r->set_path(path, no_subresource_cache);
```

Comment:
This is likely not a good idea, If you are loading without cache I would simply not set the path. The problem is, of course, the internal resources because in Godot 3 they are loaded into the cache, so I think they would need to be set into some sort of `Map<subindex,resource>` and loaded from there.

Reply:
Would something like:
```c++
uint32_t index = f->get_32();
String path = res_path + "::" + itos(index);
IntResource sub{ path, index };
RES res;
if (internal_resources_cache.has(sub)) {
	res = internal_resources_cache[sub];
}
else {
	res = ResourceLoader::load(path, "", no_subresource_cache);
	internal_resources_cache.insert(sub, res);
}
```
work suitably?

Reply:
I think I would just have something like `Map<int,RES>` for internal resources and use this always, but only load it from the cache in the main function when it first parses it.